pkgs/top-level/python-packages.nix

```diff
@@ -4196,6 +4196,28 @@ in modules // {
       description = "pytest plugin with mechanisms for caching across test runs";
     };
   };
+  
+  pytestdjango = buildPythonPackage rec {
+    name = "pytest-django-${version}";
+    version = "2.9.1";
+
+    src = pkgs.fetchurl {
+      url = "https://pypi.python.org/packages/source/p/pytest-django/${name}.tar.gz";
+      sha256 = "1mmc7zsz3dlhs6sx4sppkj1vgshabi362r1a8b8wpj1qfximpqcb";
+    };
+    
+    # doing this to allow depending packages to find
+    # pytest's binaries
+    pytest = self.pytest;
```

Comment:
Ok so here's somewhere I use it - in a django project "package", which depends on its main code, `xyzCode`, which itself depends on a whole host of things, which it then itself exposes as attrs. One of those is pytestdjango. Then I can do:

```
installPhase = ''
  ...
  cp ${xyzCode.pytestdjango.pytest}/bin/.py.test-wrapped $out/bin/py.test
  ...
'';
```

Copying the actual `py.test` binary into the _project_ outpath causes `wrapPythonPrograms` to re-wrap it with the _project's_ `PYTHONPATH` etc. values, meaning that we've created a `py.test` command which, _whatever_ context it runs in, will _always_ be run with the project's standard paths setup. None of these odd _but it works one way when I'm running this from `manage.py` and another when invoked from `py.test`_ problems.